rules/otbn.bzl

```diff
@@ -67,6 +67,15 @@ def _otbn_binary(ctx, additional_srcs = []):
     We achieve by having `otbn_build.py` emit a conventional RV32I library
     that other rules can depend on in their `deps`.
     """
+    binary = getattr(ctx.attr, "binary", None)
+    if binary != None:
```

Comment:
Should this be `is not None`? (I don't actually know anything about .bzl files, but I assume this is just Python, and would suggest following the usual convention)

Similarly for `_run_sim_test` below.